### PR TITLE
feat(cli): improve project templates to be compatible with `yarn`

### PR DESCRIPTION
**Pull request by frbuceta**
## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- API Documentation in code was updated
- Documentation in [/docs/site](../tree/master/docs/site) was updated
- Affected artifact templates in `packages/cli` were updated
- Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by frbuceta** [comment]
Do not merge this PR please, I have more things to change

**Comment by frbuceta** [comment]
You can check, thanks for waiting

**Comment by raymondfeng** [comment]
@frbuceta Can we have some test coverage for the proposed changes? Please also fix https://travis-ci.com/github/strongloop/loopback-next/jobs/346356545

**Comment by frbuceta** [comment]
> @frbuceta Can we have some test coverage for the proposed changes? Please also fix https://travis-ci.com/github/strongloop/loopback-next/jobs/346356545

Yeah let me have some free time

File: packages/cli/generators/project/templates/package.json.ejs [hunk 1]
```diff
@@ -1,11 +1,14 @@
 {
   "name": "<%= project.name -%>",
-  "version": "1.0.0",
+  "version": "0.0.1",
```
Comment (bajtos) [inline packages/cli/generators/project/templates/package.json.ejs:3]: What's wrong with `1.0.0` in the scaffolded project? This change seems rather unrelated to yarn support to me 🤷🏻

Comment (raymondfeng) [reply packages/cli/generators/project/templates/package.json.ejs:3]: Let's keep `0.0.1` as is.

Comment (frbuceta) [reply packages/cli/generators/project/templates/package.json.ejs:3]: Before this time that I was missing I had already spoken with @raymondfeng to set the version to 0.0.1. I should still separate it into a different commit.

**Comment by dhmlau** [comment]
We just switch the contribution method from CLA to DCO, making your contribution easier in the future. Please sign the commits with DCO by amending your commit messages with `-s` flag and push the changes again. If you're using the command line, you can:

```
git commit --amend -s
git push --force-with-lease
```

Please refer to [this docs page](https://loopback.io/doc/en/contrib/code-contrib.html#developer-certificate-of-origin-dco) for details. Thanks!

**Comment by frbuceta** [comment]
> We just switch the contribution method from CLA to DCO, making your contribution easier in the future. Please sign the commits with DCO by amending your commit messages with `-s` flag and push the changes again. If you're using the command line, you can:
> 
> ```
> git commit --amend -s
> git push --force-with-lease
> ```
> 
> Please refer to [this docs page](https://loopback.io/doc/en/contrib/code-contrib.html#developer-certificate-of-origin-dco) for details. Thanks!

Done

**Comment by dhmlau** [comment]
@frbuceta, thanks for your contribution again. Your PR has landed! 🎉
